fingerprint/models.py

```diff
@@ -54,9 +54,20 @@ def __str__(self) -> str:
         return self.value
 
     @classmethod
-    @lru_cache(maxsize=1024)
     def from_value(cls, value: str) -> Url:
-        return cls.objects.get_or_create(value=value)[0]
+        cache_key = getattr(settings, 'FINGERPRINT_URL_CACHE_KEY', 'fingerprint_url')
+
+        if not cache_key:
+            return cls.objects.get_or_create(value=value)[0]
+
+        cache_key = f'{cache_key}_{value}'
```

Comment:
just hardcode prefix here

If someone is worried about key collisions, they can use different cache entirely, even one that would just set their own, even more unique key prefix ( https://docs.djangoproject.com/en/5.0/ref/settings/#std-setting-CACHES-KEY_PREFIX )

This app may at some point have multiple different types of things to cache. End user should not need to configure N new settings value just because of that.